CryptKeeperZK/crypt-keeper-extension
# chore: add dotenv package and support debug env variable

## Explanation

Add env variables support for debug variables

## More Information

Fixes #112

## Screenshots/Screencaps

N/A

## Manual Testing Steps

1. Create `.env` file (use `.env.example` structure)
2. Try to change `CRYPTKEEPER_DEBUG` values.
3. See the console output

## Pre-Merge Checklist


### + If there are functional changes:


> PR template source from [github.com/MetaMask](https://github.com/MetaMask)
